src/storage/src/compaction/task.rs

```diff
@@ -175,20 +176,21 @@ impl CompactionOutput {
         let output_file_id = FileId::random();
         let opts = WriteOptions {};
 
-        let SstInfo {
-            time_range,
-            file_size,
-        } = sst_layer
+        Ok(sst_layer
             .write_sst(output_file_id, Source::Reader(reader), &opts)
-            .await?;
-
-        Ok(FileMeta {
-            region_id,
-            file_id: output_file_id,
-            time_range,
-            level: self.output_level,
-            file_size,
-        })
+            .await?
+            .map(
+                |SstInfo {
```

Comment:
What about implementing `From<SstInfo>` for `FileMeta`? Then we can simplify the code as:
```rust
.await?
.map(Into::into)
```

Reply:
> What about implementing `From<SstInfo>` for `FileMeta`? Then we can simplify the code as:
> 
> ```rust
> .await?
> .map(Into::into)
> ```

`FileMeta` also needs `region_id`, `file_id` and `level`, which `SstInfo` cannot provide.